packages/flutter/lib/src/material/page.dart

```diff
@@ -164,9 +164,6 @@ class _CupertinoBackGestureController extends NavigationGestureController {
 /// The entrance transition for the page slides the page upwards and fades it
 /// in. The exit transition is the same, but in reverse.
 ///
-/// [MaterialApp] creates material page routes for entries in the
-/// [MaterialApp.routes] map.
-///
```

Comment:
I moved it to the subclass, since MaterialApp creates instances of the subclass now.